VirtoCommerce.Platform.Web/Cache/RedisConnectionManager2.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
Please revert this file completely

Reply:
This will allow eliminating sonar errors.

Reply:
It is done